cns/restserver/restserver.go

```diff
@@ -132,7 +132,7 @@ type httpRestServiceState struct {
 	OrchestratorType                 string
 	NodeID                           string
 	Initialized                      bool
-	ContainerIDByOrchestratorContext map[string]string          // OrchestratorContext is key and value is NetworkContainerID.
+	ContainerIDByOrchestratorContext map[string]Set             // OrchestratorContext is the key and value is NetworkContainerIDs separated by comma.
```

Comment:
Set by itself won't store the values as comma separated. Remove the comma separated comment.
You can mention that in the cns state file, it will be stored as comma separated